dali/operators/expressions/arithmetic_test.cc

```diff
@@ -119,6 +137,38 @@ TEST(ArithmeticOpsTest, GetTiledCover) {
   EXPECT_EQ(std::get<1>(result1), range1);
 }
 
+namespace {
+
+template <typename T>
+std::enable_if_t<std::is_integral<T>::value, T> GenerateData(int sample, int element) {
+  static std::mt19937 gen(42);
+  std::uniform_int_distribution<> dis(-1024, 1024);
+  auto result = dis(gen);
+  return result == 0 ? 1 : result;
+}
+
+template <typename T>
+std::enable_if_t<!std::is_integral<T>::value, T> GenerateData(int sample, int element) {
+  static std::mt19937 gen(42);
+  std::uniform_real_distribution<float> dis(-1024, 1024);
+  auto result = dis(gen);
+  return result == 0.f ? 1.f : result;
```

Comment:
Add a comment about _why_ you don't allow zeros...

Reply:
done